docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: ./
+      dockerfile: ./Dockerfile.development
+    command: bash -c "/app/bin/wait-for-it.sh db:5432 -- ./bin/download_geolite2.sh;./bin/runsslserver.sh"
+    links:
+      - db
+    networks:
+      - public_nw
+    volumes:
+      - .:/app
+    environment:
+      - DATABASE_URL=postgres://postgres:postgres@db/normandy
+    ports:
+      - "8000:8000"
+
+  db:
+    restart: always
+    image: postgres:9.6
+    volumes:
+      - db_volume:/var/lib/postgresql
+    ports:
+      - "5432:5432"
```

Comment:
to avoid the port in use issue i think you could also just do something like:
```
  db:
    restart: always
    image: postgres:9.6
    volumes:
      - db_volume:/var/lib/postgresql
    networks:
      - private_nw
```

Reply:
You would also need to put app in the network `private_nw`.

Reply:
Yep this is the same pattern we use on experimenter to prevent collisions, we should do the same here.